jsonschema-validation.xml

```diff
@@ -627,6 +627,26 @@
                 </t>
             </section>
 
+            <section title='"examples"'>
+                <t>
+                    The value of this keyword MUST be an array.
+                    There are no restrictions placed on the values within the array.
+                </t>
+                <t>
+                    This keyword can be used to provide sample JSON values associated with a
+                    particular schema, for the purpose of illustrating usage.  It is
+                    RECOMMENDED that these values be valid against the associated schema.
+                </t>
+                <t>
+                    Implementations MAY use the value of "default", if present, as
+                    an additional example.  If "examples" is absent, "default"
+                    MAY still be used in this manner.
+                </t>
+                <t>
+                    This keyword MAY be used in root schemas, and in any subschemas.
+                </t>
```

Comment:
Does this paragraph serve any purpose? It's describing the default behavior for any keyword.